.gitignore

```diff
@@ -140,3 +140,5 @@ cython_debug/
 # Pycharm
 .idea
 
+# image data
+data/images/*
```

Comment:
![image](https://user-images.githubusercontent.com/44261822/126676678-b2fdc479-f401-4a85-8df9-7c1c061a54b2.png)

I actually had this previously in the gitignore. But faced the same problem. Currently I can think of two fixes.

1. Keep the images folder
2. check if the file exists or not. If not, then create the file(and folder) first

Second one seems better imo. You think you can handle that?